march_gain_scheduling/src/march_gain_scheduling/unequal_length_error.py

```diff
@@ -0,0 +1,5 @@
+import rospy
+
+class UnequalLengthError(Exception):
```

Comment:
When creating a customized error it is nice to add some additional information:
```suggestion
class UnequalLengthError(Exception):
    def __init__(self, msg=None):
        """Error class for the unequal length situation

        :param msg:
            The message that needs to be displayed.
        """
        if msg is None:
            msg = 'current_gains and needed_gains do not have the same length' 
         super(UnequalLengthError, self).__init__(msg)
```

Reply:
`UnequalLengthError` implies it is generic across all arrays, yet its message is specific for current and needed gains

Reply:
I am going to leave the exception empty. A message is passed with more explanation when it is called.